statement.md

```diff
@@ -0,0 +1,84 @@
+# JupyterLab Accessibility Statement
+
+## Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+
+## ⚠️ Draft Accessibility Statement for JupyterLab
+
+This is an accessibility statement from <span class="basic-information organization-name">Jupyter accessibility contributors</span>.
+
+### Measures to support accessibility
+
+Jupyter accessibility contributors take the following measures to ensure accessibility of JupyterLab:
+
+* Include accessibility as part of our mission statement.
+* Provide continual accessibility training for our community.
+* Assign clear accessibility goals and responsibilities.
+* Employ formal accessibility quality assurance methods.
+
+### Conformance status
+
+The <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines (WCAG)</a> defines requirements for designers and developers to improve accessibility for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA. JupyterLab is non conformant with WCAG 2.1 level AA. Non conformant means that the content does not conform the accessibility standard.
```

Comment:
Is this the auto-generated text? 

> Non conformant means that the content does not conform the
Using the word in the definition which makes it hard to follow/understand

Reply:
Stephanie Walter has an amazing illustration for this in https://stephaniewalter.design/blog/5-illustrations-to-understand-and-promote-accessibility/

Could we add this to add some visual sugar - of course with due acknowledgement and x-linking to the post

Reply:
It is the auto-generated text; most of this is not my work alone.

I'll look into how I need to add the image with credit and add it.